application/src/main/java/core/application/users/controller/UserController.java

```diff
@@ -57,8 +56,8 @@ public ApiResponse<Message> login() {
     @Operation(summary = "회원가입")
     @PostMapping("/signup")
     public ApiResponse<MessageResponseDTO> singUp(@Valid @RequestBody UserRequestDTO userRequestDTO) {
-        MessageResponseDTO msg = userService.signup(userRequestDTO);
-        return ApiResponse.onCreateSuccess(msg);
+        MessageResponseDTO messageResponseDTO = userService.signup(userRequestDTO);
+        return ApiResponse.onCreateSuccess(messageResponseDTO);
```

Comment:
응답에 `String message`만 들어간다면, api 패키지 내에 Message 객체 사용해주시면 될 것 같아요! 이전에 응답처리할 때 message만 보내주는 경우, `Message`로 통일 시켰습니다!
```suggestion
        return ApiResponse.onCreateSuccess(Message.createMessage("/*Message*/"));
```